jdk_8_maven/em/embedded/rest/blogapi/src/main/resources/data.sql

```diff
@@ -1,3 +1,7 @@
+LOCK TABLES `roles` WRITE;
+INSERT INTO `roles` VALUES (1,'ROLE_ADMIN'),(2,'ROLE_USER');
+UNLOCK TABLES;
+
```

Comment:
shouldn't the files in external modified the same way? `jdk_8_maven/em/external/rest/blogapi/src/main/resources`

Reply:
updated.